crates/ubrn_bindgen/src/bindings/react_native/gen_typescript/templates/macros.ts

```diff
@@ -237,19 +237,19 @@
     {%- endif %}
 {%- endmacro %}
 
-{%- macro async(func) %}
+{%- macro is_async(func) %}
```

Comment:
This renaming and the ones below are required because rinja doesn't allow Rust keywords as macro names. I'm not sure if the names are particularly great but this follows what uniffi-rs itself does.